core/ui/vault/settings/backup/index.tsx

```
@@ -0,0 +1,73 @@
+import { ListItem } from '@lib/ui/list/item'
+import { Modal } from '@lib/ui/modal'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { CoreView } from '../../../navigation/CoreView'
+import { useCoreNavigate } from '../../../navigation/hooks/useCoreNavigate'
+import { useCurrentVault } from '../../state/currentVault'
+import { BackupOption } from './BackupOption'
+
+const backupOptions = ['device', 'server'] as const
+type BackupOption = (typeof backupOptions)[number]
```

Biome (2.1.2)

[error] Shouldn't redeclare 'BackupOption'. Consider to delete it or rename it.

'BackupOption' is defined here:


(lint/suspicious/noRedeclare)